README.md

```diff
@@ -43,21 +44,20 @@ Precompiled bleeding-edge package builds of gRPC `master` branch's `HEAD` are up
 
 Contributions are welcome!
 
-Please read [How to contribute](CONTRIBUTING.md) which will guide you through the entire workflow of how to build the source code, how to run the tests and how to contribute your changes to
-the gRPC codebase.
+Please read [How to contribute](CONTRIBUTING.md) which will guide you through the entire workflow of how to build the source code, how to run the tests and how to contribute your changes tothe gRPC codebase.
 The document also contains info on how the contributing process works and contains best practices for creating contributions.
 
 # Troubleshooting
 
-Sometimes things go wrong. Please check out the [Troubleshooting guide](TROUBLESHOOTING.md) if you are experiencing issues with gRPC.
+If you find out there are errors or issues, feel free to check out the [Troubleshooting guide](TROUBLESHOOTING.md) if you are experiencing issues with gRPC.
```

Comment:
Change to "If you are experiencing issues with gRPC, please checkout the [Troubleshooting guide](TROUBLESHOOTING.md)."